apps/edxapp/vars/all/main.yml

```
@@ -15,6 +15,8 @@ edxapp_sql_dump_url: "https://gist.githubusercontent.com/jmaupetit/76fe7db4a8314
 edxapp_secret_name: "edxapp-{{ edxapp_vault_checksum | default('undefined_edxapp_vault_checksum') }}"
 
 # i18n
+# Should we download and compile latest translations?
+edxapp_i18n_should_update: false
```

Maybe start with `should` then the verb?
```suggestion
edxapp_should_update_18n: false
```

---

Ah ah. I knew this was a bad name (talked about it with @lunika), but I kind like namespacing vars as `{{ application.name}}_{{ scope }}_{{ purpose }}`. But I can ignore this implicit rule and update this. WDYT?

---

I would say namespacing with application name is fine. Scope may be too much since we have only a few settings?

---

Ok then, I'll do this.